src/main/java/core/basesyntax/MainApp.java

```diff
@@ -12,5 +12,25 @@
  * - Отправить все машини на работу и остановить работу всех машин.
  */
 public class MainApp {
+    public static void main(String[] args) {
 
+        Track track = new Track();
```

Comment:
What will happen if we write:
Machine track = new Track();
and the same with other objects?

How do you think, why do we create abstract class Machine?
And what us the difference using Bulldozer or Machine?